applications/openstack/cinder/cinder_file_ownership/rule.yml

```diff
@@ -32,5 +32,5 @@ rationale: ""
 severity: unknown
 
 references:
-    nist: FOO-1(a)
     cui: 3.1.5
+    nist: FOO-1(a)
```

Comment:
The things that surface when we touch every rule in the project...
I think these are bogus references, couldn't find them o official doc.

Reply:
:D

Reply:
100% Bogus reference

Reply:
There weren't that many, I've opened https://github.com/ComplianceAsCode/content/pull/6917.